code/game/objects/items/storage/dakis.dm

```diff
@@ -0,0 +1,52 @@
+//////////////////////////////////
+ //dakimakuras
+ //////////////////////////////////
+
+/obj/item/storage/daki
+	name = "dakimakura"
+	desc = "A large pillow depicting a girl in a compromising position. Featuring as many dimensions as you."
+	icon = 'icons/obj/daki.dmi'
+	icon_state = "daki_base"
+	lefthand_file = "icons/mob/inhands/misc/daki_lefthand.dmi"
+	righthand_file = "icons/mob/inhands/misc/daki_righthand.dmi"
+	slot_flags = SLOT_BACK
+	storage_slots = 3
+	w_class = WEIGHT_CLASS_NORMAL
+	var/cooldowntime = 20
+	var/static/list/dakimakura_options = list("Callie","Casca","Chaika","Elisabeth","Foxy Grandpa","Haruko","Holo","Ian","Jolyne","Kurisu","Marie","Mugi","Nar'Sie","Patchouli","Plutia","Rei","Reisen","Naga","Squid","Squigly","Tomoko","Toriel","Umaru","Yaranaika","Yoko") //Kurisu is the ideal girl." - Me, Logos. //how could someone make something so good then say some retarded shit like this
```

Comment:
These all need to be SS13 related and you need way better sprites

Reply:
i am willing to rework all of these but i can't really that right now as i'm leaving for a vacation very very very soon

Reply:
so i'd rather get this in then when i'm back edit them to be thematic!

Reply:
Maybe whichever spriter you bribe can PR your code for you as well then, if it was merely bad sprites I'd go with it but this is a pretty big violation of the reference policy etc

Reply:
you're lucky i care about dakis more than i do my trip, i'll swap them out either today or tomorrow!